src/Core/src/Configuration.php

```diff
@@ -56,6 +59,10 @@ final class Configuration
             self::OPTION_WEB_IDENTITY_TOKEN_FILE => 'AWS_WEB_IDENTITY_TOKEN_FILE',
             self::OPTION_ROLE_SESSION_NAME => 'AWS_ROLE_SESSION_NAME',
         ],
+        [
+            self::OPTION_ECS_AUTH_ENDPOINT => 'AWS_CONTAINER_CREDENTIALS_RELATIVE_URI',
+            self::OPTION_METADATA_SERVICE_TIMEOUT => 'AWS_METADATA_SERVICE_TIMEOUT',
```

Comment:
Im not sure these two should be in a group. 

@jderusse you have better understanding of this.

Reply:
Agreed with @Nyholm, a "group" means: if, at least, one option is defined by the user, the others env variables will be ignored.
ie, if user setup `accessKeyId`, and `accessKeySecret` but leave `sessionToken` empty, it will not be override by `AWS_SESSION_TOKEN`.

I think we can use `OPTION_METADATA_SERVICE_TIMEOUT` in several place,I think this variable shoud be moved  in a dedicated "group"

Reply:
Got it, I wasnt sure about this but now is clear as crystal